src/c/core/session/session.c

```diff
@@ -254,7 +254,8 @@ bool uxr_buffer_performance(uxrSession *session,
     payload.buf = buf;
     payload.len = len;
     ucdrBuffer mb;
-    if (uxr_prepare_stream_to_write(&session->streams, stream_id, (uint16_t)(SUBHEADER_SIZE + 8 + len), &mb))
+    const uint16_t size_to_write = (uint16_t)(SUBHEADER_SIZE + sizeof(epoch_time) + len);
+    if (uxr_prepare_stream_to_write(&session->streams, stream_id, size_to_write, &mb))
     {
         uint8_t flags = (echo) ? UXR_ECHO : 0;
         (void) uxr_buffer_submessage_header(&mb, SUBMESSAGE_ID_PERFORMANCE, (uint16_t)(8 + len), flags);
```

Comment:
Could be modify the `8` to `sizeof(epoch_time)` to be according to the upper change?

Reply:
Ok perfect, I changed it.